### PR TITLE
Responsive full view

### PR DESCRIPTION
**Pull request by mcallegari10**
## Description

Add full screen in responsive mobile view

## Screenshot

image

Landscape:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,29 +102,13 @@ class App extends Component {
 export default App;
 ```
 
-4- Customize the widget to match your app design! You can add both props to manage the title of the widget, the avatar it will use and of course the styles it will have.
+4- Customize the widget to match your app design! You can add both props to manage the title of the widget, the avatar it will use and of course, you can change, from the CSS, the styles the widget will have.
```
Comment (SKOLZ) [inline README.md]: there are a lot of commas at the end of this sentence, check how to paraphrase it with iniaki

File: README.md [hunk 2]
```diff
@@ -156,91 +141,78 @@ export default App;
...
 
+- **handleNewUserMessage:** (PropTypes.func.isRequired) Function to handle the user input, will receive the full text message when submitted
 - **title:** (PropTypes.string) Title of the widget
 - **subtitle:** (PropTypes.string) Subtitle of the widget
 - **senderPlaceHolder:** (PropTypes.string) The placeholder of the message input
-- **profileAvatar:** (PropTypes.string.isRequired) The profile image that will be set on the responses
-- **stylesInjected:** (Proptypes.object) The styles to be injected, see [styles](#styles)
-- **handleNewUserMessage:** (PropTypes.func.isRequired) Function to handle the user input, will receive the full text message when submitted
+- **profileAvatar:** (PropTypes.string) The profile image that will be set on the responses
+- **showCloseButton:** (PropTypes.bool) Show or not the close button in full screen mode
```
Comment (SKOLZ) [inline README.md]: show or hide

File: src/components/Widget/components/Conversation/components/Header/index.js [hunk 3]
```diff
@@ -1,17 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import close from 'assets/clear-button.svg';
 import './style.scss';
 
-const Header = ({ title, subtitle }) =>
+const Header = ({ title, subtitle, toggleChat, showCloseButton }) =>
   <div className="header">
+    {showCloseButton &&
```
Comment (SKOLZ) [inline src/components/Widget/components/Conversation/components/Header/index.js]: leave the `{` in a separate line

File: src/components/Widget/components/Conversation/components/Header/index.js [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
+      <button className="close-button" onClick={toggleChat}>
+        <img src={close} className="close" alt="" />
```
Comment (SKOLZ) [inline src/components/Widget/components/Conversation/components/Header/index.js]: change the alt value to "close"

File: src/components/Widget/components/Conversation/style.scss [hunk 5]
```diff
@@ -10,3 +10,11 @@
...
   @include animation(0, 0.5s, slide-out);
 }
+
+@media screen and (max-width: 800px) {
+  .conversation-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+}
```
Comment (SKOLZ) [inline src/components/Widget/components/Conversation/style.scss]: empty line at EOF

File: src/components/Widget/components/Conversation/components/Header/index.js (hunk 3 above)
Comment (mcallegari10) [reply src/components/Widget/components/Conversation/components/Header/index.js]: Mmm.. I don't really like the way it looks writing in a separate line and leaving the `{` alone :disappointed:

**Pull request merged**